docs/contributing.rst

```diff
@@ -11,20 +11,44 @@ Contributing to Scrapy
 
 There are many ways to contribute to Scrapy. Here are some of them:
 
-* Report bugs and request features in the `issue tracker`_, trying to follow
-  the guidelines detailed in `Reporting bugs`_ below.
 
-* Submit patches for new functionalities and/or bug fixes. Please read
-  :ref:`writing-patches` and `Submitting patches`_ below for details on how to
-  write and submit a patch.
+**Types of Contributions**
 
-* Blog about Scrapy. Tell the world how you're using Scrapy. This will help
+We love to get your help, suggestions, and contributions. There are many ways to
+contribute to this project.
+
+The first thing to do, regardless of how you'd like to contribute to the project,
+is to check out our Code of Conduct and to keep that in mind while interacting
+ with other contributors and users.
+
+* Non-Programming
+
+ * Blog about Scrapy. Tell the world how you're using Scrapy. This will help
   newcomers with more examples and will help the Scrapy project to increase its
   visibility.
 
+  * Documentation
+  Improve the documentation `Scrapy Documentation <https://docs.scrapy.org/en/latest/>`_ 
+  by opening a new issue on the Issues tab of the Github repo. 
+  Use the "blank issue" option for documentation change suggestions.
```

Comment:
As I said earlier, this is wrong. Documentation change suggestions should use the same options as other change suggestions.

Because of this I'm still not convinced documentation even needs a separate section here.